### PR TITLE
Feature/search

### PR DESCRIPTION
**Pull request by janimuhlestein**
Fixed User.js data issues.

### CODE REVIEW

File: client/src/utils/GlobalState.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const StoreProvider = ({ value = [], ...props }) => {
     cartOpen: false,
     categories: [],
     currentCategory: '',
+    selectedProducts: ''
   });
```
Comment (jesseparent) [inline client/src/utils/GlobalState.js:15]: I think you need selectedUser here

**Review by jesseparent — APPROVED** [review]
File is being overwritten

File: client/src/utils/GlobalState.js (hunk 1 above)
Comment (janimuhlestein) [reply client/src/utils/GlobalState.js:15]: I agree. Changed.

**Comment by janimuhlestein** [comment]
Closing this one and will do another.

**Pull request closed without merging**